origin.spec

```diff
@@ -413,6 +418,10 @@ fi
 
 %post node
 %systemd_post %{name}-node.service
+# If accounting is not currently enabled systemd reexec
+if [[ `systemctl show mananager | grep -q -e CPUAccounting=no -e MemoryAccounting=no` == 0 ]]; then
```

Comment:
I changed it to check for docker and our node service to avoid possible collisions with another service that may have explicitly disabled accounting. I've not found a better way to figure out the real default.